src/Rules/Generators/YieldInGeneratorRule.php

```diff
@@ -44,9 +45,7 @@ public function processNode(Node $node, Scope $scope): array
 		} elseif ($scopeFunction !== null) {
 			$returnType = ParametersAcceptorSelector::selectSingle($scopeFunction->getVariants())->getReturnType();
 		} else {
-			return [
-				'Yield can be used only inside a function.',
-			];
+			return [RuleErrorBuilder::message('Yield can be used only inside a function.')->line($node->getLine())->build()];
```

Comment:
You can leave RuleErrorBuilder in (since given enough time, I want to all rules return this instance), but remove the line() method call.